packages/dev/core/src/Inputs/scene.inputManager.ts

```diff
@@ -865,47 +875,49 @@ export class InputManager {
         this._deviceSourceManager.onDeviceConnectedObservable.add((deviceSource) => {
             if (deviceSource.deviceType === DeviceType.Mouse) {
                 deviceSource.onInputChangedObservable.add((eventData) => {
-                    if (eventData.inputIndex === PointerInput.LeftClick || eventData.inputIndex === PointerInput.MiddleClick || eventData.inputIndex === PointerInput.RightClick) {
-                        const evt = eventData as IPointerEvent;
-                        if (attachDown && deviceSource.getInput(evt.inputIndex) === 1) {
-                            this._onPointerDown(evt);
-                        } else if (attachUp && deviceSource.getInput(evt.inputIndex) === 0) {
-                            this._onPointerUp(evt);
+                    if ((eventData.inputIndex === PointerInput.LeftClick || eventData.inputIndex === PointerInput.MiddleClick || eventData.inputIndex === PointerInput.RightClick)) {

```

Comment:
Did this line get an extra set of parens?

Reply:
Yeah, looks like it was a possible copy-paste error.  Removed parens